### ISSUE TITLE
How to interpret each row of configuration data in txprofiles.conf?

### ISSUE THREAD

**Issue by lida2003** [OpenIPC/adaptive-link#10]
Is there any test or theory on these numbers?

https://github.com/sickgreg/OpenIPC-Adaptive-Link/blob/3cc9d3ac00d1024db34d9450d8426b3f9b43c86b/txprofiles.conf#L1-L6

**Comment by sickgreg** [comment]
There is a lot of theory about all of those to learn about. There is a basic description in readme.md.

Each line is a link speed / network settings profile that gets chosen based on link quality

**Comment by lida2003** [comment]
Yeah, quite a lot of params. As I said, we are trying to tune or use proper params for our FPV wfb card.

We have feedback on txpower=58 definitely cause PA saturation. That's why I write this post to ask when I saw the above table line4 txpower=58. And txpower is below 45 in latest `txprofiles.conf`.

Did you do the test you self or got those data from somewhere, if there is any link avaliable?

https://github.com/sickgreg/OpenIPC-Adaptive-Link/blob/39cfc2dc3c368b79deedd564e5613822459b0ee9/txprofiles.conf#L1-L11

**Comment by sickgreg** [comment]
I did a lot of flight testing myself and fly almost every day.  Have never scoped the signal or anything.  I don't have that equipment.  I got further range with higher numbers.  Also more heat, of course. There may have been more fec_recovered packets with higher power numbers but i still seemed to be able to fly further into the forest.

In the new version, i have backed them off based on 1,  i wanted compatible numbers with 8812AU-AF1 card, 2, heat and 3, hearing the same information about saturation as you did.  I heard numbers above 46 cause problems. On MCS4 and above i can confirm a lot of problems with high powers.  Not so sure about it when using low mcs
